lib/limax.js

```diff
@@ -6,14 +6,24 @@ var hepburn = require('hepburn');
 
 module.exports = function(text, opt) {
   var options = opt || {};
+  var separateNumbers = true;
+
   if (typeof options === 'string') {
     options = {separator: options};
   }
 
+  if (typeof options.separateNumbers !== 'undefined') {
```

Comment:
If you want to enforce a boolean, yes. With !== undefined you can put in any truthy or falsy value.

Feel free to change it.